packages/dev-server-swc/package.json

```diff
@@ -21,7 +21,7 @@
     }
   },
   "engines": {
-    "node": ">=14.0.0"
+    "node": ">=12.0.0"
```

Comment:
why?

Reply:
To match all other packages in the repo. I was worried that the tests were failing because of the change but I can totally revert this.